Themes/default/scripts/jquery.sceditor.smf.js

```diff
@@ -647,7 +647,9 @@ $.sceditor.plugins.bbcode.bbcode.set(
 			if (typeof attrs.date !== "undefined" && attrs.date)
 			{
 				attr_date = attrs.date;
-				sDate = '<date timestamp="' + attr_date + '">' + new Date(attrs.date * 1000) + '</date>';
+				tDate = new Date(attrs.date * 1000);
```

Comment:
It just is inconsistent to declare a local variable and use it in one instance and not use it in another.